ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -445,7 +446,7 @@ class WebSocketService(
       .map { case (oeso, ejv) =>
         for {
           offPrefix <- oeso.sequence
-          jv <- ejv
+          jv <- ejv.widenLeft[Error]
```

Comment:
I'm also wondering whats up with the widening here :see_no_evil:

Reply:
You know, it might be worth adding another implicit to `widenLeft` et al to ensure that it isn't a no-op, i.e. it will fail to compile if the source type already equals the goal type. What do you think?